x-pack/plugins/observability_solution/apm/server/routes/traces/get_trace_items.ts

```diff
@@ -107,12 +114,6 @@ export async function getTraceItems({
         ERROR_EXCEPTION,
```

Comment:
these need to use `_source`

Reply:
I am a bit confused here, in the previous [comment](https://github.com/elastic/kibana/pull/192608/files#r1772913451) the suggestion was to not use fields and source together - should we implement a logic to merge the fields from both?

Reply:
yes - my comment was around fetching the same fields from `fields` and `_source`. For span links and exceptions, I think you need to merge from both.